CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1 @@
+- Fix: bug setting throttling to -1 in the subscription cache for v2 subscriptions (Issue #2114)
```

Comment:
It would be better to have a description closer to what the bug means from the user perspective (have a look to https://github.com/telefonicaid/fiware-orion/issues/2114#issuecomment-216587273 in order to understand what I mean).